diff -Nur conntrack-tools-0.9.10/src/mcast.c conntrack-tools-0.9.10/src/mcast.c
--- conntrack-tools-0.9.10/src/mcast.c	2009-01-26 11:21:51.472204316 +0000
+++ conntrack-tools-0.9.10/src/mcast.c	2009-01-26 11:15:00.000000000 +0000
@@ -29,6 +29,7 @@
 #include <sys/ioctl.h>
 #include <net/if.h>
 #include <errno.h>
+#include <limits.h>
 
 struct mcast_sock *mcast_server_create(struct mcast_conf *conf)
 {
diff -Nur conntrack-tools-0.9.10/src/sync-mode.c conntrack-tools-0.9.10/src/sync-mode.c
--- conntrack-tools-0.9.10/src/sync-mode.c	2009-01-26 11:21:57.201787444 +0000
+++ conntrack-tools-0.9.10/src/sync-mode.c	2009-01-26 11:15:26.000000000 +0000
@@ -29,6 +29,7 @@
 #include "queue.h"
 
 #include <errno.h>
+#include <limits.h>
 #include <unistd.h>
 #include <time.h>
 #include <string.h>
